### DISCUSSION TITLE
Kibana Dark Mode

### DISCUSSION THREAD

**Discussion by imalc3142** [Security-Onion-Solutions/securityonion#6285] [Unsupported Versions] [answered]
Is there any way to permanently turn dark mode in Kibana off?  

I suffer from migraines and dark mode is a trigger.  

Right now every time I go into Kibana or open a new Kibana window I need to go to advance settings and change the mode.  It would make SecurityOnion much more usable for me if I could change the theme permanently.

Thanks,

**Comment by dougburks** [comment, marked as answer]
Our upcoming 2.3.90 release should allow you to change the dark mode setting permanently:
https://github.com/Security-Onion-Solutions/securityonion/projects/26
